requirements.txt

```diff
@@ -27,3 +27,4 @@ rocksdb
 cffi
 protobuf
 colorama
+unicorn==2.1.0
```

Comment:
Why do we need unicorn at all?

Reply:
Avatar2 needs unicorn
`from avatar2 import Avatar, TargetStates` in `firmwire.py` line 20
To be able to work with arm ach, avatar2 does from .arm import *
`File "/usr/local/lib/python3.8/dist-packages/avatar2/archs/__init__.py", line 2, in <module>
    from .arm import *`
To then `from unicorn import *`

I redirect you to https://github.com/FirmWire/FirmWire/actions/runs/11369406000/job/31626827653 if you need more details

An easy fix I found is downgrading unicorn to 2.1.0

Reply:
Thanks for pointing this out - I'm currently working on making the required changes to avatar2